website_crm_quick_answer/data/base_automation_data.xml

```diff
@@ -7,7 +7,7 @@
             <field name="model_id" ref="crm.model_crm_lead"/>
             <field name="use_default_to" eval="True"/>
             <field name="email_from">${object.company_id.email}</field>
-            <field name="lang">${object.env.context.get("lang")}</field>
+            <field name="lang">${object.env.context.get("lang") or object.partner_id.lang or object.company_id.lang or object.env.user.lang}</field>
```

Comment:
`object.company_id.lang` is a valid field or a usable one?

Reply:
Yes it's valid.

https://github.com/odoo/odoo/blob/f6e14195e58c3e4837e82df950a92a33e470dd2a/addons/crm/models/crm_lead.py#L132